### PR TITLE
[tls] Fix validate ovn service cert secret

### PR DESCRIPTION
**Pull request by Deydra71**
Add OVN cert validation to detect changes in `cert-neutron-ovndbs`. This patch includes the OVN cert hash in the combined TLS secrets hash.

### CODE REVIEW

File: controllers/neutronapi_controller.go [hunk 1]
```diff
@@ -491,7 +491,43 @@ func (r *NeutronAPIReconciler) reconcileInit(
...
+			secretVars["ovnCertsHash"] = env.SetValue(ovnCertsHash)
+		}
+	}
+
+	// Combine all cert hashes into a single hash
+	combinedCertsHash, err := util.HashOfInputHashes(secretVars)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+	secretVars[tls.TLSHashName] = env.SetValue(combinedCertsHash)
```
Comment (stuggi) [inline controllers/neutronapi_controller.go:530]: I don't think we need this. bellow in L539 there is a call for `r.createHashOfInputHashes()` which calcs the hash of hashes

Comment (Deydra71) [reply controllers/neutronapi_controller.go]: Right, I thought about keeping the original logic with `tls.TLSHashName` but it's redundant to calculate hashes two times.

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**